jlh/WeatherApp/app/src/main/res/layout/item_forecast.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="horizontal"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
I think you meant a `wrap_content` attribute value😆.
![screenshot_20170321-161254](https://cloud.githubusercontent.com/assets/4919888/24154422/7254df8e-0e51-11e7-8d64-207fa8a5657b.png)